.ko.yaml

```diff
@@ -0,0 +1,10 @@
+---
+builds:
+  - id: imgctrl
+    dir: .
```

Comment:
`dir:` is `.` by default, so we could just remove this.

Reply:
Sure!